lib/modules/datasource/maven/readme.md

```diff
@@ -0,0 +1,12 @@
+### Fetching changelogs from API
+In case your repository doesn't hold a `CHANGELOG.md` file and you need to fetch changelogs from an API (just like the one from [Gitlab](https://docs.gitlab.com/ee/api/releases) or [Github](https://docs.github.com/en/rest/releases)), you need to configure `Maven` so that `renovatebot` is able to determine the `sourceUrl` when fetching info about the artifact.
```

Comment:
```suggestion

In case your repository doesn't hold a `CHANGELOG.md` file and you need to fetch changelogs from an API (just like the one from [Gitlab](https://docs.gitlab.com/ee/api/releases) or [Github](https://docs.github.com/en/rest/releases)), you need to configure `Maven` so that `renovatebot` is able to determine the `sourceUrl` when fetching info about the artifact.
```

Add new line.